spanner/api/Spanner.Samples.Tests/AddColumnAsyncPostgresTest.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright 2022 Google Inc.
```

Comment:
@Rishabh-V Do you think the version should be 2023 in all files ?

Reply:
I think it is fine to keep the original year, since we are just renaming the files.